deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -2665,6 +2665,31 @@ fun(Conf) ->
     list_to_binary(cuttlefish:conf_get("amqp1_0.default_vhost", Conf))
 end}.
 
+{mapping, "rabbit.stream_replication_port_range.min", "osiris.port_range", [
```

Comment:
Do we really need this `rabbit.` prefix? How about `stream.replication.port_range.min` and `stream.replication.port_range.max`? `stream.*` is a suffix we already use in `rabbitmq_stream`.